test/re/frida/MethodTest.java

```diff
@@ -519,6 +521,21 @@ public void nativeMethodCanBeReplaced() {
         assertEquals("randomString", script.getNextMessage());
     }
 
+    @Test
+    public void genericArrayTypeShouldBePreserved() {
+        loadScript("var GenericArray = Java.use('re.frida.GenericArray');" +
+                "var getArray2 = GenericArray.getArray2;" +
```

Comment:
Let's avoid `var` for new tests. (Would be good to port our existing tests to modern JavaScript someday.)